src/test/java/nl/topicus/mssql2monetdb/test/CopyTableTest.java

```diff
@@ -0,0 +1,48 @@
+package nl.topicus.mssql2monetdb.test;
+
+import static org.junit.Assert.*;
+import nl.topicus.mssql2monetdb.CopyTable;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CopyTableTest {
+
+	@Test
+	public void testGenerateCountQuery() {
+		CopyTable table = new CopyTable();
+		
+		// test standard count query based on table from name
+		table.setFromName("test");
+		Assert.assertEquals(table.generateCountQuery(), "SELECT COUNT(*) FROM [test]");
+		
+		table.setFromName(null);
+		
+		// test a custom count query
+		String customCountQuery = "SELECT COUNT(*) FROM myQuery WHERE bla = 'something'";
+		table.setFromCountQuery(customCountQuery);		
+		Assert.assertEquals(table.generateCountQuery(), customCountQuery);	
```

Comment:
Dat klopt maar dat is toch eigenlijk ook wat ik wil testen, namelijk dat de `generateCountQuery` methode exact hetzelfde terug geeft als wat ik opgeef i.p.v. iets anders? Voor een getter+setter zou dat een vrij suffe test zijn maar in dit geval doet de `generateCountQuery` methode wel iets meer waarbij ik verwacht dat als er een eigen FromCountQuery wordt geset dat deze ook exact hetzelfde terugkomt. Of is het in dat geval nog steeds best-practice om het verwachte resultaat te hardcoden?